README.md

```diff
@@ -1,16 +1,65 @@
 # test-operator
 This is a test operator that automates testing of an OpenStack deployment
-running on a Kubernetes cluster. The operator runs
-[Tempest](https://opendev.org/openstack/tempest) tests.
+running on a OpenShift cluster. The operator supports execution of the following
+tests:
+
+- [Tempest](https://opendev.org/openstack/tempest)
+
+- [Tobiko](https://tobiko.readthedocs.io/en/0.4.7/index.html)
+
+- [Horizon tests](https://review.opendev.org/openstack/horizon)
+
+- Arbitrary tests written as an Ansible role (refer to Ansibletest CR)
 
 
 ## Getting Started
-First, you will need an OpenStack Platform running on OpenShift. See, the
-[ci-framework documentation](https://ci-framework.readthedocs.io/en/latest/)
+First, you will need an OpenStack Platform running on top of OpenShift. See:
+
+- [ci-framework documentation](https://ci-framework.readthedocs.io/en/latest/) or
+
+- [install_yamls GitHub repository](https://github.com/openstack-k8s-operators/install_yamls/blob/main/README.md)
+
 to get you started. It will get you through the installation of such environment.
 
 Then proceed to our [documentation](https://openstack-k8s-operators.github.io/test-operator/).
 
+## Contributing
+Please, make sure that pre-commit checks pass prior to proposing a PR.
+
+1. Install pre-commit tool
+
+```bash
+python3 -m venv .venv
+. .venv/bin/activate
+python3 -m pip install pre-commit
+pre-commit install --install-hooks
+```
+
+2. Execute pre-commit checks
+
+```bash
+pre-commit run --show-diff-on-failure --color=always --all-files --show-diff-on-failure --verbose
+```
+
+### Test It Out
+Run the following commands if you want to quickly test your test-operator related
+changes locally.
+
+1. Install the CRDs into the cluster:
+
+```sh
+make install
+```
+
+2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
```

Comment:
Maybe add another step from the line in brackets? Or create a note? The brackets have a long sentence in it, so it doesn't look that readable, but that's just minor detail.

Reply:
Thank you! I wrapped the line for better readability.